.env.example

```diff
@@ -0,0 +1,2 @@
+SOURCECRED_DISCORD_TOKEN=TjkyOTk1OFg1NTA2MTQ4NgVw.TqpnSA.fWIrGvOaQuT8Z-5-8dN-hdChkub
+SOURCECRED_GITHUB_TOKEN=913043df20670ec28e4428eacc80218a1bcce0dd
```

Comment:
are these real tokens? (shipping a real token for the discord test instance could be nice, as potentially could shipping a permissionless github token from a sandboxed user account, altho ppl would likely forget to set their own token and eventually run into rate limit issues from sharing the sandbox token)

Reply:
No, they are not real tokens. I think its best to get users to upload their real tokens. I know Discord for sure scans public git repos for exposed tokens and disables them proactively since a lot of people make this mistake

Reply:
Since they're not real tokens, I recommend making that a lot more obvious by using something that may resemble a real token, but clearly isn't. Or even something like:

`SOURCECRED_GITHUB_TOKEN=YourTokenHere123`